rmc/utils/constraint.py

```diff
@@ -760,6 +769,200 @@ def process_sections(
     return ht
 
 
+def get_rescue_1break_transcripts(
+    overwrite: bool, initial_threshold: float = 6.6, rescue_threshold: float = 5.0
+) -> Tuple[hl.expr.SetExpression]:
+    """
+    Get transcripts that have a single breakpoint above the 'rescue' threshold.
+
+    These transcripts did not have a breakpoint that was over the initial search threshold
+    but have a breakpoint that is above the lower 'rescue' search threshold.
+
+    .. note::
+        This function assumes that `process_sections` was run with `save_full_chisq_ht`
+        set to True during the initial single break search (round 1).
+
+    :param overwrite: Whether to overwrite output data if it exists.
+    :param initial_threshold: Chi square significance threshold associated with
+        the initial search pathway.
+    :param rescue_threshold: Lower chi square significance threshold associated with
+        the 'rescue' search pathway.
+    :return: None; function writes HT to resource path.
+    """
+    # Read in the simultaneous breaks results from initial search round 1
+    simul_results_path = simul_search_round_bucket_path(
+        is_rescue=True,
+        search_num=1,
+        bucket_type="final_results",
+    )
+    simul_ht = hl.read_table(
+        f"{simul_results_path}/merged.ht",
+    )
+    simul_ht = simul_ht.annotate(transcript=simul_ht.section.split("_")[0])
```

Comment:
that's true! the function returns the transcripts downstream, so this seemed more intuitive, but happy to update if you feel strongly about this

Reply:
just changed it to convert sections to transcripts within `get_rescue_transcripts_and_create_no_breaks_ht`

Reply:
Hmmmm actually ykw I think it does make more sense for it to be transcripts. I was thinking about the minimum operations necessary since these functions aren't used anywhere else but that isn't the most intuitive given the function names. Sorry about that!